btrdb/stream.py

```diff
@@ -1599,8 +1599,12 @@ def earliest(self):
             lambda s: s.nearest(start, version=versions.get(s.uuid, 0), backward=False),
             self._streams,
         )
-        for point, _ in earliest_points_gen:
-            earliest.append(point)
+        for point in earliest_points_gen:
+            if point is not None:
+                earliest.append(point[0])
+            else:
+                # add RawPoint with None as timestamp and value
+                earliest.append(RawPoint(None, None))
```

Comment:
What's the reason to add an empty `RawPoint` to `earliest` instead of appending nothing? Is it so we get some point for each stream? Even if it's blank?